src/ring_theory/adjoin/basic.lean

```diff
@@ -205,6 +205,28 @@ le_antisymm
     (λ p ⟨n, hn⟩ hp, by rw [alg_hom.map_mul, mv_polynomial.aeval_def _ (mv_polynomial.X _),
       mv_polynomial.eval₂_X]; exact subalgebra.mul_mem _ hp (subset_adjoin hn)))
 
+lemma _root_.mv_polynomial.range_aeval {σ : Type*} (f : σ → A) : 
```

Comment:
This lemma is spliced between two lemmas with names of the form `adjoin.*_eq_range`. I think it would be good to follow that naming pattern (and the direction of the statement).

Reply:
So perhaps `adjoin_range_eq_range`?

Reply:
I did it like this because I thought this particular version was more likely to be used in this direction, the lhs seems more likely to turn up in your goal than the rhs. For that reason I want `range_aeval` to be part of the name so it's easy to find.

Reply:
Perhaps the compromise is simply to have both for discoverability?

Reply:
At any rate, `adjoin_singleton_eq_range` is exactly analogous to this lemma but with `polynomial` instead of `mv_polynomial`, so you should probably add `polynomial.range_aeval` too if you keep `mv_polynomial.range_aeval`

Reply:
I changed the name